application/src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentController.java

```diff
@@ -26,16 +28,20 @@
 import uk.gov.hmcts.dm.service.DocumentContentVersionService;
 
 import javax.annotation.PostConstruct;
+import javax.servlet.http.HttpServletResponse;
 import javax.validation.Valid;
+import java.io.IOException;
 import java.util.List;
 import java.util.UUID;
 
+import static java.lang.String.format;
+import static org.apache.commons.lang.StringUtils.isBlank;
+
```

Comment:
Shall we use the later org.apache.commons.lang3

Reply:
Will change